app/templates/home.hbs

```diff
@@ -140,10 +140,16 @@
     </div>
 
     {{else}}
-        {{add-layer index=index layers=model.theme.layers editMode=editMode isDragging=isDragging}}
+        <div  class="col-xs-3">
+
+        </div>
+        <div class="{{holderCSSEditMode}}" style="padding:0">
+            <br><br>
+            {{add-layer index=index layers=model.theme.layers editMode=editMode type=type isDragging=isDragging percentage=percentage}}
+        </div>
     {{/if}}
 
-    {{layer-footer guid=model.guid editMode=editMode holderCSSEditMode=holderCSSEditMode}}
+    {{layer-footer guid=model.guid editMode=editMode holderCSSEditMode=holderCSSEditMode percentage=percentage mainPercentage=mainPercentage}}
```

Comment:
I'm assuming this is to do with the horizontal resize; How come this needs to know about the percentage?

Reply:
it is outside the layer wrapper thus needing to be there